initializr-service/src/main/resources/application.yml

```diff
@@ -91,6 +91,28 @@ initializr:
     kotlin:
       version: 1.1.2-5
   dependencies:
+    - name: Azure
+      content:
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          version: 0.1.0
```

Comment:
What's the suggested way about the starter version management? Is that we need to update the version number here when we release a new version?